tests/cdi/cdi_test.go

```diff
@@ -36,11 +37,28 @@ var _ = Describe("Importing and starting a VM using CDI", func() {
 
 		It("the VM should be created successfully", func() {
 			tests.ProcessTemplateWithParameters(rawVMFilePath, dstVMFilePath, "VM_NAME="+vmName, "PVC_NAME="+pvcName, "VM_APIVERSION="+vmAPIVersion)
-			tests.CreateResourceWithFilePathTestNamespace("vmi", vmName, dstVMFilePath)
+			tests.CreateResourceWithFilePathTestNamespace(dstVMFilePath)
 		})
 
 		Specify("the VM should be running", func() {
 			tests.WaitUntilResourceReadyByNameTestNamespace("vmi", vmName, "-o=jsonpath='{.status.phase}'", "Running")
 		})
 	})
+
+	Context("PVC with invalid image url will be failed", func() {
+		It("clean up the importer pod", func() {
+			tests.DeleteResourceWithLabelTestNamespace("pod", "app=containerized-data-importer")
```

Comment:
this label should be a const rather than hard-coded several times in these tests. If CDI issue #239 is addressed then this label will change.